airbyte-integrations/connectors/source-file/build.gradle

```
@@ -8,6 +10,11 @@ airbytePython {
     moduleDirectory 'source_file'
 }
 
+task("customIntegrationTesPython", type: PythonTask, dependsOn: installTestReqs){
+    module = "pytest"
+    command = "-s integration_tests"
+}
+
```

This runs these tests as part of `./gradlew :airbyte-integrations:connectors:source-file:integrationTest` which is what CI runs for integration tests
```suggestion

integrationTest.dependsOn("customIntegrationTestPython")

```